conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
I think the original resolution is 2048x3320 over the 180 degree FOV but the user can select the size of the crop and subsampling they like using these settings.